packages/arb-token-bridge-ui/src/components/App/AppContext.tsx

```diff
@@ -99,6 +111,10 @@ export const useAppContextActions = (dispatchOverride?: Dispatch<Action>) => {
     dispatch({ type: 'layout.set_txhistory_panel_visible', payload: true })
   }
 
+  const setShowSentTransactions = (payload: boolean) => {
```

Comment:
I think maybe something like `showSentTransactions` and `showReceivedTransactions` are a bit nicer